runtime/interface.go

```diff
@@ -56,6 +66,13 @@ type EmptyRuntimeInterface struct{}
 func (i *EmptyRuntimeInterface) ResolveImport(location Location) ([]byte, error) {
 	return nil, nil
 }
+func (i *EmptyRuntimeInterface) GetCachedProgram(location Location) (*ast.Program, error) {
+	return nil, errors.New("cache not implemented")
```

Comment:
I think this should not return an error: if there's nothing cached, that's not an error:

```suggestion
	return nil, nil
```

Reply:
Agreed 👍

Reply:
I was originally planning to use that for bubbling up cache errors, going to maintain this signature as per your following comment.